Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -104,6 +105,24 @@ public enum TypeDescription: Codable, Hashable {
       } else {
         return "\(parentType.asSource).\(name)<\(generics.map { $0.asSource }.joined(separator: ", "))>"
       }
+    case let .attributed(type, specifier: specifier, attributes: attributes):
+      func attributesFromList(_ attributes: [String]) -> String {
+        attributes
+          .map { "@\($0)" }
+          .joined(separator: " ")
+      }
+      switch (specifier, attributes) {
+      case let (.some(specifier), .none):
+        return "\(specifier) \(type.asSource)"
+      case let (.none, .some(attributes)):
+        return "\(attributesFromList(attributes)) \(type.asSource)"
+      case let (.some(specifier), .some(attributes)):
+        // This case likely represents an error.
+        return "\(specifier) \(attributesFromList(attributes)) \(type.asSource)"
+      case (.none, .none):
+        // This case likely represents an error.
```

Comment:
This case, however, I'm pretty convinced is an error. I don't know how we'd end up as an attributed type without either a specifier or attributes

Reply:
Note that because I believe this is an error I haven't been able to write a test case that covers this case.

Reply:
Let's assert?

Reply:
Added an assert in two spots for good measure, and avoided surfacing the type we think is a bug at the parsing stage:  5b44aac3dc93f45ed9300f45752a194e975d58c8